templates/Layout/ProductPage.ss

```diff
@@ -22,19 +22,24 @@
                     <p>
                         <strong>Base Price:</strong> $Price.Nice<br>
                         <strong>Weight:</strong> $Weight lbs<br>
-                        <strong>Code:</strong> $Code
+                        <strong>Code:</strong> $Code<br>
+                        $ProductRating
                     </p>
                     <div class="content">$Content</div>
 
-
             </article>
-            $PageComments
         </div>
     </div>
 
     <div class="sidebar unit size1of4">
         $PurchaseForm
     </div>
+
+</div>
+<div class="line">
```

Comment:
Does `<% if $ProductReviews %>` make sense so we don't have empty div blocks (if product reviews isn't installed)? or are the blocks needed regardless for spacing?